plotly/plotly/plotly.py

```diff
@@ -680,7 +680,14 @@ def get(figure_or_data, format='png', width=None, height=None, scale=None):
                 raise exceptions.PlotlyError("The response "
                                              "from plotly could "
                                              "not be translated.")
-            raise exceptions.PlotlyError(return_data['error'])
+
+            error_msg = return_data.get('error')
+            if error_msg is None:
+                error_msg = return_data.get('errors')
+            if error_msg is None:
```

Comment:
You can avoid the extra if statement by setting 
Line 688 as the default value in the get method.